libcontainer/process_linux.go

```diff
@@ -43,12 +43,20 @@ type parentProcess interface {
 	externalDescriptors() []string
 
 	setExternalDescriptors(fds []string)
+
+	getChildLogs()
```

Comment:
I don't really like this name -- maybe `forwardChildLogs` or `proxyChildLogs` to make it clear that this method actually pipes directly to logrus.